universal-login-commons/lib/integration/ethereum/walletFromBackupCode.ts

```diff
@@ -0,0 +1,29 @@
+import {utils, Wallet} from 'ethers';
+import scrypt from 'scrypt-js';
+
+export function walletFromBackupCodes(username: string, password: string): Promise<Wallet> {
```

Comment:
Come on, no single test? 

You refactor security fragile function, I think it is better to just copy-paste in the same form, rather than accidentally introduce bug.

Reply:
We should also stick to fromBrainWallet name, it is a significant name